measures/ResStockArguments/measure.rb

```diff
@@ -523,12 +539,76 @@ def run(model, runner, user_arguments)
 
     # Seasons
-    if args[:use_auto_heating_season] && args[:hvac_control_heating_season_period] == Constants.Auto
+    if (!args[:use_auto_heating_season] && args[:hvac_control_heating_season_period].include?('Unavailable')) || (!args[:use_auto_cooling_season] && args[:hvac_control_cooling_season_period].include?('Unavailable'))
+      heating_months, cooling_months, sim_calendar_year = get_heating_and_cooling_seasons(args, epw_file, weather)
+    end
+
+    if args[:use_auto_heating_season]
       args[:hvac_control_heating_season_period] = HPXML::BuildingAmerica
+    else
+      if args[:hvac_control_heating_season_period].include?('Unavailable')
+        begin_month, begin_day, end_month, end_day = Schedule.get_begin_and_end_dates_from_monthly_array(heating_months, sim_calendar_year)
+        begin_day_num = Schedule.get_day_num_from_month_day(sim_calendar_year, begin_month, begin_day)
+        end_day_num = Schedule.get_day_num_from_month_day(sim_calendar_year, end_month, end_day)
+
+        if args[:hvac_control_heating_season_period] == 'Unavailable 1 Day'
+          unavail_begin_day_num, unavail_end_day_num = get_subset_begin_end_day_num(args[:building_id], 1, begin_day_num, end_day_num, sim_calendar_year)
+        elsif args[:hvac_control_heating_season_period] == 'Unavailable 1 Week'
+          unavail_begin_day_num, unavail_end_day_num = get_subset_begin_end_day_num(args[:building_id], 7, begin_day_num, end_day_num, sim_calendar_year)
+        elsif args[:hvac_control_heating_season_period] == 'Unavailable 1 Month'
+          unavail_begin_day_num, unavail_end_day_num = get_subset_begin_end_day_num(args[:building_id], 30, begin_day_num, end_day_num, sim_calendar_year)
+        elsif args[:hvac_control_heating_season_period] == 'Unavailable 2 Weeks'
+          unavail_begin_day_num, unavail_end_day_num = get_subset_begin_end_day_num(args[:building_id], 14, begin_day_num, end_day_num, sim_calendar_year)
+        elsif args[:hvac_control_heating_season_period] == 'Unavailable 3 Days'
+          unavail_begin_day_num, unavail_end_day_num = get_subset_begin_end_day_num(args[:building_id], 3, begin_day_num, end_day_num, sim_calendar_year)
+        elsif args[:hvac_control_heating_season_period] == 'Unavailable 3 Months'
+          unavail_begin_day_num, unavail_end_day_num = get_subset_begin_end_day_num(args[:building_id], 90, begin_day_num, end_day_num, sim_calendar_year)
+        elsif args[:hvac_control_heating_season_period] == 'Unavailable All Days'
+          # FIXME: this would set the season equal to all days EXCEPT for the BA season, which is technically not what we want. but there is no value we can pass in to mean year round unavailable; only achievable using HVAC unavailable periods.
+          unavail_begin_day_num, unavail_end_day_num = get_subset_begin_end_day_num(args[:building_id], Constants.NumDaysInYear(sim_calendar_year), begin_day_num, end_day_num, sim_calendar_year)
```

Comment:
Alternatives may be:
* set a 1-day heating season (e.g., Jan 1 or in the middle of summer)
* set FractionHeatLoadServed=0

Reply:
Is another option to just set heating/cooling system to "none"?

Reply:
"The second doesn't allow the space temperature to float" -- I'm pretty sure it should. Can you share a file where you're seeing otherwise?

Reply:
I thought there was an ideal air system when fraction < 0, no?

Reply:
I think that comment was written not considering a system with FractionLoadServed=0 since that's pretty unusual. It should more explicitly say < 1 and > 0 (e.g., room ACs serving a portion of the load).

Reply:
Interesting. So you're suggesting that I could use FractionHeatLoadServed=0 instead of a heating season of Jul 1 - Jul 1 for a broken heating system scenario?

Reply:
Fixing that comment [here](https://github.com/NREL/OpenStudio-HPXML/pull/1788/commits/81b7658c730780f47e7b7296c46bcd07363a36db).

Reply:
> Interesting. So you're suggesting that I could use FractionHeatLoadServed=0 instead of a heating season of Jul 1 - Jul 1 for a broken heating system scenario?

I don't know what behavior you're looking for, but I think it'd be no different than not having that HVAC system altogether.

Reply:
Well, there might still be some ancillary uses like crankcase heating.